lib/cartopy/tests/mpl/test_examples.py

```diff
@@ -29,19 +29,14 @@ def new_fn(*args, **kwargs):
 
 
 @pytest.mark.natural_earth
-@ExampleImageTesting(['global_map'],
-                     tolerance=4.5 if MPL_VERSION < '2' else 0.5)
+@ExampleImageTesting(['global_map'], tolerance=0.5)
 def test_global_map():
     import cartopy.examples.global_map as example
     example.main()
 
 
-if MPL_VERSION < '2':
-    contour_labels_tolerance = 7.5
-elif MPL_VERSION <= '2.0.2':
-    contour_labels_tolerance = 1.24
-elif MPL_VERSION <= '2.1.2':
-    contour_labels_tolerance = 0.63
+if MPL_VERSION < '3':
+    contour_labels_tolerance = 9.8
```

Comment:
Confused why this went up.

Reply:
I have no idea either, hence my statement about the oddities at the beginning... I'll attach the difference image showing the comparison and why I had to increase it.
![result-contour_label-failed-diff](https://user-images.githubusercontent.com/12417828/87552191-92cf9000-c66e-11ea-80b5-c452fbbb47fa.png)

When you were testing for 0.18 did you test on the latest MPL 2.x? I'm pretty sure we were missing that from our test suite coverage on CI, so we were testing against 1.x and 3.x only.

Reply:
I tested every single possible combination of Matplotlib (latest micro only), Proj, and Python, as available in conda.

These are text issues, so FreeType-related. Did you use Matplotlib from the `conda-forge/label/testing` channel to get the one with FreeType 2.6? It looks like they may only have 2.2.2, not 2.2.5 though.

Reply:
On my local environment that I produced that image with I have
`freetype                  2.10.0               he06d7ca_0    conda-forge`
On the CI build it looks like it is pulling from conda-forge (not the /label/testing) and getting a more recent version.
`freetype                  2.10.2               he06d7ca_0    conda-forge`

conda won't let me pin `freetype=2.6` for any of the 2.2.x series of MPL either.

Reply:
Yeah, we only have 2.2.2 on `conda-forge/label/testing`. You really want to use the versions from there since they're statically linked to freetype 2.6.1, which is what matplotlib uses (currently at least) for their static images for testing. Any reason you made 2.2.3 the minimum on Travis? Could we just make it 2.2.2?

Reply:
I can't install that locally?
```
conda create -n test
conda install --channel conda-forge/label/testing matplotlib=2.2.2
```
gives me freetype 2.10.2 still... Maybe because I'm on OSX though? Explicitly, it completely fails if I add:
`conda install --channel conda-forge/label/testing matplotlib=2.2.2 freetype=2.6`

Just saw 2.2.3 listed on the original Issue and went with it. Maybe we shouldn't even pin to bugfix release versions?

Reply:
Matplotlib from that channel is statically linked with FreeType; the other package version is not important.

Reply:
Doh! I missed that even though you both said it :(

Reply:
Hm... It doesn't want to pull from that channel.
Here is what CI grabbed:
`matplotlib                2.2.2            py37hb69df0a_2    defaults`

I'm also curious why it is only this one image failing here too and not others with text...

Reply:
Ahh, so I just looked at the testing channel and they only have py36 wheels built for 2.2.2.
https://anaconda.org/conda-forge/matplotlib/files?version=2.2.2&channel=testing
So, I'm not sure what the best option here is. We could add an additional py36 test to the matrix I suppose.

Reply:
Looks like using Py36 solved that problem. I think this is good for a full review now. Leaving the test matrix at Py36 with minimum dependencies and Py38 with latest everything seems reasonable enough for now.